ainft/chat/messages.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import json
+from typing import List
+
+from ain.ain import Ain
+
+from ..types import (
+    SetOperation,
+    SetMultiOperation,
+    TransactionInput,
+    Message,
+    MessageTransactionResult,
+)
+from ..utils import *
+
+
+class Messages:
+    _max_content_length = 200
```

Comment:
could you feedback on limiting message content length to `200` characters?
if content is exceed max length, it will be automatically trimmed.

Reply:
I think it's fine if we can apply the same logic to ainft-js.

Reply:
yes, I'll apply this logic to ainft-js too! thanks!